standalone/src/recording/components/recording_view/component.tsx

```diff
@@ -0,0 +1,299 @@
+            }
+
+            const MuteIcon = isMuted ? MutedIcon : UnmutedIcon;
+
+            return (
+                <li
+                    key={'participants_profile_' + idx}
+                    style={{display: 'flex', flexDirection: 'column', justifyContent: 'center', alignItems: 'center', margin: '16px'}}
+                >
+
+                    <div style={{position: 'relative'}}>
+                        <Avatar
+                            size={50}
+                            fontSize={18}
+                            border={false}
+                            borderGlow={isSpeaking}
+                            url={this.props.pictures[profile.id]}
+                        />
+                        <div
+                            style={{
+                                position: 'absolute',
+                                display: 'flex',
+                                justifyContent: 'center',
+                                alignItems: 'center',
+                                bottom: 0,
+                                right: 0,
+                                background: 'rgba(50, 50, 50, 1)',
+                                borderRadius: '30px',
+                                width: '20px',
+                                height: '20px',
+                            }}
+                        >
+                            <MuteIcon
+                                fill={isMuted ? '#C4C4C4' : '#3DB887'}
+                                style={{width: '14px', height: '14px'}}
+                                stroke={isMuted ? '#C4C4C4' : ''}
+                            />
+                        </div>
+                        <div
+                            style={{
+                                position: 'absolute',
+                                display: isHandRaised ? 'flex' : 'none',
+                                justifyContent: 'center',
+                                alignItems: 'center',
+                                top: 0,
+                                right: 0,
+                                background: 'rgba(50, 50, 50, 1)',
+                                borderRadius: '30px',
+                                width: '20px',
+                                height: '20px',
+                                fontSize: '12px',
+                            }}
+                        >
+                            {'✋'}
+                        </div>
+                    </div>
+
+                    <span style={{fontWeight: 600, fontSize: '12px', margin: '8px 0'}}>
+                        {getUserDisplayName(profile)}
+                    </span>
+
+                </li>
+            );
+        });
+    }
+
+    render() {
+        const callsClient = this.getCallsClient();
+        if (!callsClient) {
+            return null;
+        }
+
+        const sharingID = this.props.screenSharingID;
+
+        return (
```

Comment:
I think I've mentioned it before, but I can't help but wonder if there's a way to use the existing component, maybe by adding some flags to its props... Otherwise this is going to be another "platform" we have to support with any UI changes. Probably not for this PR, but maybe soon?

Reply:
Yeah, I touched on this a bit before but I felt there was too much divergence to make it work through the existing component. The end result of that would have been worse in my mind as it required adding a layer of conditional rendering logic on top of an already overly complex component. 

What I am trying to do instead (in the upcoming PRs) is to share more sub-components between the two views so that changes on one side are reflected on the other side without having to duplicate code and work.